internal/statesync/snapshotter.go

```diff
@@ -3,13 +3,12 @@ package statesync
 import (
```

Comment:
we can remove this custom snapshotter if statesync works in v0.47 just need to check